packages/bot-web-ui/src/components/load-modal/load-modal.scss

```diff
@@ -166,3 +175,12 @@
         margin: 0;
     }
 }
+@media screen and (max-width: 768px) {
```

Comment:
is this related to mobile screen, did not follow the use case so asking

Reply:
Yes @vinu-deriv Bhai. According to the design, the banner needs some spacing on top and bottom but based on our existing modal design it's different on mobile and desktop so needed to add the media query.